recipes/tacker_server.rb

```diff
@@ -45,12 +45,10 @@ class ::Chef::Recipe
 
 #------------------------------------------------------------------------------
 apt_update ''
-package 'git'
 package 'python-pip'
```

Comment:
all packages should be installed in an each loop (cosmetics)

Reply:
Done.